backend/utils/sidebar.py

```diff
@@ -34,7 +34,9 @@ async def __get_room_members(
                             }
         """
         room_members = room.get("room_members")
-        room_members.pop(member_id, "not-found")  # remove self from room members
+
+        if len(list(room_members.keys())) > 1 and room.get("room_type") != RoomType.DM:
```

Comment:
@o4codes  You don't actually need .keys() again if you're using list. You can write it write is as list(room_members). Its the same thing.

This should also be 'and room.get("room_type") != RoomType.CHANNEL' so that it factors in GroupDMs and DMs
```suggestion
        if len(list(room_members)) > 1 and room.get("room_type") != RoomType.CHANNEL:
```

Reply:
Thanks
@pauline-banye  fixed